src/converter/one/convert/JfrConverter.java

```diff
@@ -21,31 +21,52 @@
 public abstract class JfrConverter extends Classifier {
     protected final JfrReader jfr;
     protected final Arguments args;
+    private final IEventAggregator leakAggregator;
```

Comment:
It feels like a leaky abstraction (pun intended): we pretend it to be some abstract aggregator, although we know it can be only MallocLeakAggregator.